packages/appium/lib/constants.js

```diff
@@ -49,6 +50,9 @@ export const KNOWN_DRIVERS = Object.freeze(
     flutter: 'appium-flutter-driver',
     safari: 'appium-safari-driver',
     gecko: 'appium-geckodriver',
+    tizentv: 'appium-tizen-tv-driver',
```

Comment:
I am not quite sure it would be fair to have these shortcuts as other developers could add their own drivers for the same platform. My proposal is to only keep shortcuts for officially supported plugins/drivers

Reply:
what then if the particular developers stops supporting a driver or an extension? It is easy to add something but hard to remove.

Yes, I agree to remove tizen and youi shortcuts

Reply:
i understand the point about downgrading official recognition as well. @boneskull @KazuCocoa what do you think?

Reply:
I'd like to keep maintained drivers/plugins as the central place here as my primary thought.
Then, maybe we can print a link to the drivers/plugins list as the output of `driver list`/`plugin list` command and keep 3rd party drivers/plugins in the documentation? Something like...:

```
$ appium driver list
✔ Listing available drivers
- xcuitest@4.13.1 [installed (NPM)]
- uiautomator2 [not installed]
- youiengine [not installed]
- windows [not installed]
- mac [not installed]
- mac2 [not installed]
- espresso [not installed]
- tizen [not installed]
- flutter [not installed]
- safari [not installed]
- gecko [not installed]
More drivers are 'url'
```

Reply:
btw, `mac` also can be removed since we mostly can maintain `mac2` for newer macOS (10.15+)